upgrade.go

```diff
@@ -74,13 +75,43 @@ func getVersionDiff(oldVersion, newversion string) (left, right string) {
 	}
 
 	if errOld == nil && errNew == nil {
-		if old.Version == new.Version {
-			left = string(old.Version) + "-" + red(string(old.Pkgrel))
-			right = string(new.Version) + "-" + green(string(new.Pkgrel))
-		} else {
-			left = red(string(old.Version)) + "-" + string(old.Pkgrel)
-			right = bold(green(string(new.Version))) + "-" + string(new.Pkgrel)
+		oldVersion := string(old.Version) + "-" + string(old.Pkgrel)
```

Comment:
The version strings are passed in, you then convert them to a CompleteVersion and back to a string. Won't this always equal the same thing that was passed in? If there is some reason to do this you should be using old.String()

Reply:
Done, thank you!